apps/filecoin-site/src/app/_styles/globals.css

```diff
@@ -110,4 +110,22 @@
   .slide-over-panel {
     @apply bg-white;
   }
+
+  /* TOOLTIP */
+  .tooltip {
+    @apply rounded-xl border border-zinc-200 bg-white text-zinc-950 shadow-sm;
+
+    .tooltip-arrow {
+      @apply fill-zinc-300;
+    }
+  }
+
+  .social-link {
+    @apply rounded-full border-2 border-zinc-200 p-1 text-zinc-950;
+  }
+
+  /* COPY TO CLIPBOARD */
+  .copy-to-clipboard-button {
+    @apply rounded-full border-2 border-zinc-200 p-1 text-zinc-950;
+  }
```

Comment:
Thought of merging it into 1 class, but we have to keep `copy-to-clipboard-button `naming, which doesn't make sense to put on social link component...

Reply:
I don't understand ..

Reply:
What about something like

```css
@layer components {
  .icon-button {
    @apply rounded-full border-2 border-zinc-200 p-1 text-zinc-950;
  }
}

.social-link {
  @apply icon-button;
  /* Additional social-link specific styles if needed */
}

.copy-to-clipboard-button {
  @apply icon-button;
  /* Additional copy button specific styles if needed */
}
```